### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'render'

### ISSUE THREAD

**Issue by mksingh2104** [mortazavilab/PyWGCNA#53]
This line of code:

pyWGCNA_m1_10x.CoexpressionModulePlot(modules=modules, numGenes=100, numConnections=1000, minTOM=0, file_name="all")

giving me following error.
AttributeError: 'NoneType' object has no attribute 'render'

What I am doing wrong?

**Comment by nargesr** [comment]
Hi,

would you mind sending the full error?

**Comment by mksingh2104** [comment]
/Users/mksingh/PyWGCNA_UpWork1//figures/network/all.html
Traceback (most recent call last):
  File "/Users/mksingh/PyWGCNA_UpWork1/network.py", line 10, in <module>
    pyWGCNA_m1_10x.CoexpressionModulePlot(modules=modules, numGenes=10, numConnections=100, minTOM=0, file_name="all")
  File "/Users/mksingh/anaconda3/lib/python3.10/site-packages/PyWGCNA/wgcna.py", line 3391, in CoexpressionModulePlot
    net.show(f"{self.outputPath}/figures/network/{file_name}.html")
  File "/Users/mksingh/anaconda3/lib/python3.10/site-packages/pyvis/network.py", line 546, in show
    self.write_html(name, open_browser=False,notebook=True)
  File "/Users/mksingh/anaconda3/lib/python3.10/site-packages/pyvis/network.py", line 515, in write_html
    self.html = self.generate_html(notebook=notebook)
  File "/Users/mksingh/anaconda3/lib/python3.10/site-packages/pyvis/network.py", line 479, in generate_html
    self.html = template.render(height=height,
AttributeError: 'NoneType' object has no attribute 'render'

**Comment by nargesr** [comment]
can you send me your pickle file?

**Comment by mksingh2104** [comment]
I am attaching all the input files as pickle file is large in size. 
[sample_info.csv](https://github.com/mortazavilab/PyWGCNA/files/12008406/sample_info.csv)
[trimmed_means_transposed.csv](https://github.com/mortazavilab/PyWGCNA/files/12008407/trimmed_means_transposed.csv)

Running script:
----
import PyWGCNA

geneExp = 'trimmed_means_transposed.csv'

pyWGCNA_m1_10x = PyWGCNA.WGCNA(name='m1_10x', species='human', geneExpPath=geneExp, outputPath='/Users/manojkumarsingh/PyWGCNA_', save=True)

print( pyWGCNA_m1_10x.geneExpr.to_df().head(5) )

pyWGCNA_m1_10x.preprocess()

pyWGCNA_m1_10x.findModules()


pyWGCNA_m1_10x.updateSampleInfo(path='/Users/manojkumarsingh/PyWGCNA_/sample_info.csv', sep=',')

# add color for metadata
pyWGCNA_m1_10x.setMetadataColor('Sex', {'F': 'green',
                                       'M': 'yellow'})
pyWGCNA_m1_10x.setMetadataColor('Genotype', {'Nothing': 'darkviolet',
                                            'Something': 'deeppink'})
pyWGCNA_m1_10x.setMetadataColor('Age', {'0': 'thistle',
                                       '1': 'purple'})
pyWGCNA_m1_10x.setMetadataColor('Tissue', {'M1': 'red',
                                          'Cortex': 'blue'})

geneList = PyWGCNA.getGeneList(dataset='hsapiens_gene_ensembl', attributes=['ensembl_gene_id', 'external_gene_name', 'gene_biotype'], maps=['gene_id', 'gene_name', 'gene_biotype'])


pyWGCNA_m1_10x.updateGeneInfo(geneList)


pyWGCNA_m1_10x.analyseWGCNA()

pyWGCNA_m1_10x.saveWGCNA()


import PyWGCNA

pyWGCNA_m1_10x = PyWGCNA.readWGCNA("m1_10x.p")

print(pyWGCNA_m1_10x.datExpr.var.head(5) )

#pyWGCNA_m1_10x.CoexpressionModulePlot(modules=["silver"], numGenes=10, numConnections=100, minTOM=0)
modules = pyWGCNA_m1_10x.datExpr.var.moduleColors.unique().tolist()
print( modules )
pyWGCNA_m1_10x.CoexpressionModulePlot(modules=modules, numGenes=10, numConnections=100, minTOM=0, file_name="all")
----

**Comment by nargesr** [comment]
Hi,

I was able to run your code without any problem and gave me what it suppose to give! The only thing I changed is the outputPath which I don't believe is related to the error you got!

I would check the version of packages you have. I have listed mine [here](https://github.com/mortazavilab/PyWGCNA/blob/main/requirements.txt)

for example this is what I got for silver module

![image](https://github.com/mortazavilab/PyWGCNA/assets/57965133/42129b38-15bf-43d6-8605-d42cee72317f)
